src/app/components/countdown/Countdown.jsx

```diff
@@ -62,20 +64,21 @@ const Countdown = ({ date }) => {
           height={isDesktop ? 400 : 300}
           numberOfPieces={50}
           tweenDuration={2}
+          colors={confettiColors}
         />
       )}
       <FlipNumbers
         play
         numbers={timer}
-        color="white"
-        width={isDesktop ? 60 : 30}
+        color="#322660"
+        width={isDesktop ? 40 : 30}
         height={isDesktop ? 50 : 30}
         nonNumberStyle={{
           fontSize: isDesktop ? "40px" : "30px"
         }}
         numberStyles={{
           textAlign: "left",
-          fontSize: isDesktop ? "20px" : "30px",
+          fontSize: isDesktop ? "30px" : "30px",
```

Comment:
i think if these ended up being the same value then we don't need the `isDesktop` check anymore, we can just leave it as "30px" as is